utils/argumentParser.js

```diff
@@ -126,7 +126,7 @@ function resolveJson(argumentsObj, matchObj) {
     }
     currentLevel[matchParts[matchParts.length-1]] = stringToValue(matchObj.value);
   } catch (e) {
-    console.log("SEVERE: Exception occurred trying to generate object from input:",e);
+    console.log("ZWED0007E - SEVERE: Exception occurred trying to generate object from input:",e);
```

Comment:
Does not match the JSON, which has:
"ZWED0007E":"RESERVED: %s invalid version %s",

Reply:
corrected.